tests/core/test_inventory.py

```diff
@@ -530,3 +530,50 @@ def test_get_hosts_dict(self, inv):
         assert "group_1" in dev1_groups
         assert dev2_paramiko_opts["username"] == "root"
         assert "dev3.group_2" in hosts_dict
+
+    def test_add_group_to_host_runtime(self, inv):
```

Comment:
excellent. Even if we remove the `add_to_group` and `remove_from_group` methods I’d like to keep this tests as it helps testing the runtime of nornir.

Reply:
👍